src/Helper/ArrayHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace YapepBase\Helper;
+
+class ArrayHelper extends HelperAbstract
+{
+    public function getElementAsInt(array $array, string $key, ?int $default = null): ?int
+    {
+        return isset($array[$key])
```

Comment:
Should this be array key exists? Do we want to handle NULLs in here?

Reply:
I was thinking about this and decided that it's safer to use isset. If we cast null values it can cause bugs as it will be casted to whatever you ask for.